android/app/src/main/res/layout/large_widget_layout.xml

```diff
@@ -0,0 +1,304 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mainLinearLayout"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="@android:color/white"
+    android:orientation="vertical"
+    android:padding="16dp">
```

Comment:
There seems to be issue that I can't position widget to device edges. There is always quite lot of margin/padding in left and right sides. I'm not sure what causes this, but maybe some padding setting like this?

Also small widget seems to have same issue.

Reply:
If you want you can a create a new ticket about this issue.

It could be disucussed with others.
If needed I can do some widget 'stretching' later.

But I would like to postpone this for now, because we are in a hurry.
OK?